example.env

```diff
@@ -10,9 +10,11 @@ OPENAI_API_VERSION=""
 
 # azure form recognizer
 AZURE_FORM_RECOGNIZER_ENDPOINT = 
+AZURE_FORM_RECOGNIZER_NAME = ""
 AZURE_FORM_RECOGNIZER_KEY = ""
 
-# azure cognitive services
+# azure cognitive search
 COGSEARCH_NAME = "" 
 COGSEARCH_INDEX_NAME = ""
 COGSEARCH_API_KEY = "" 
+COGSEARCH_ADDRESS = ""
```

Comment:
Would you be opposed to using the cogsearch endpoint similar to the other code? If not, we will want to be consistent in the e2e sample

service_endpoint = "https://"+config["COGSEARCH_NAME"] + ".search.windows.net"

Reply:
good point!